packages/appsync-emulator-serverless/schemaTest.js

```diff
@@ -21,21 +22,15 @@ const setupDocumentDB = async (serverlessConfig, appSyncConfig, dynamodb) => {
       {},
     );
 
-  const { dataSources = [] } = appSyncConfig;
-  const appSyncTables = dataSources.filter(
-    source => source.type === 'AMAZON_DYNAMODB',
-  );
-
   const list = await Promise.all(
-    appSyncTables.map(async ({ config: { tableName } }) => {
+    Object.values(resourceTables).map(async ({ Properties: props }) => {
```

Comment:
small but notable change... we now create *all* tables listed instead of just those used by appasync.